utils_cv/detection/dataset.py

```diff
@@ -256,6 +289,43 @@ def show_ims(self, rows: int = 1, cols: int = 3) -> None:
         """
         plot_grid(display_bboxes, self._get_random_anno, rows=rows, cols=cols)
 
+    def show_im_transformations(
```

Comment:
Nice! We could demonstrate this in on of our notebooks, with an explanation when not to use e.g. horizontal flipping.